Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.ShowCase.ts2

```diff
@@ -0,0 +1,22 @@
+/**
+* NodeTypes around Show Cases
+*/
+
+
+prototype(Neos.NeosIo:Reference.ShowCase.Document) < prototype(Neos.NeosIo:Reference.Document) {
+	body {
+		content {
+			main = Neos.NeosIo:Reference.ShowCase
```

Comment:
Not sure if we should use another Protoytype and actually style it? This page is kinda hidden to the user since we don't link to it but it would still by acessible, right? Right now it looks kind of broken ?! Also the name of the reference type has some issues. But it does work in the list view. What do you think @bjen? Spend some love and some basic sylings or maybe redirect the list or something?!

<img width="1409" alt="screen shot 2016-09-09 at 21 46 48" src="https://cloud.githubusercontent.com/assets/769789/18400624/06e2c908-76d7-11e6-9b18-ee0417a6355a.png">